src/main/java/com/annapolisworks/OpNode.java

```diff
@@ -0,0 +1,27 @@
+package com.annapolisworks;
+
+public class OpNode {
+
+    public OpNode lastNode;
+    public OpNode nextNode;
+    public Operator operator;
+    public double lastNumber;
+    public double nextNumber;
+
+
+    //constructor
+    OpNode(OpNode myLastNode, OpNode myNextNode, Operator myOperator, double myLastNumber, double myNextNumber) {
```

Comment:
1. You don't need to comment your constructor.
2. Your constructor is package level.
3. You have an extra new line between the variables and the constructor.